frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -1529,19 +1591,27 @@ impl<T: Config> Pallet<T> {
 		//   system
 		// - signed phase was complete or not started, in which case finalization is idempotent and
 		//   inexpensive (1 read of an empty vector).
+		//
+		// If the election fails, `fn emergency_throttling` checks if EPM should enter in the
+		// emergency phase. The emergency phase is throttled if not enough "electing" (i.e. signed
+		// phase and unsigned phase) blocks have passed since last successful election.
 		let _ = Self::finalize_signed_phase();
 		<QueuedSolution<T>>::take()
 			.ok_or(ElectionError::<T>::NothingQueued)
-			.or_else(|_| {
-				T::Fallback::instant_elect(None, None)
-					.map_err(|fe| ElectionError::Fallback(fe))
-					.and_then(|supports| {
-						Ok(ReadySolution {
-							supports,
-							score: Default::default(),
-							compute: ElectionCompute::Fallback,
+			.or_else(|err| {
+				if !Self::emergency_phase_throttling() {
```

Comment:
name suggestion
```suggestion
				if !Self::fallback_throttling() {
```

Reply:
Answered to this renaming in the comment above.